vulture_os/toolkit/api_parser/cybereason/cybereason.py

```diff
@@ -136,235 +133,113 @@ def execute_query(self, method, url, query, timeout=10):
-            malop["timestamp"] = float(datetime.datetime.now().timestamp())
-        try:
-            malop["timestamp_closed"] = float(malop['closeTime']) / 1000
-        except Exception as e:
-            msg = f"Error enriching timestamp closed: {e}"
-            logger.error(f"[{__parser__}]:parseTimestamps: {msg}",
-                         extra={'frontend': str(self.frontend)})
-            malop["timestamp_closed"] = 0.0
+    def get_appliance_version(self):
+        version_url = f"{self.host}/{self.APP_VERSION_URI}"
         try:
-            malop["timestamp_start"] = float(malop['malopStartTime']) / 1000
+            ret = self.execute_query("GET", version_url, {})
+            return True, ret['data']['version']
         except Exception as e:
-            msg = f"Error enriching timestamp start: {e}"
-            logger.error(f"[{__parser__}]:parseTimestamps: {msg}",
-                         extra={'frontend': str(self.frontend)})
-            malop["timestamp_start"] = 0.0
-
-        return malop
-
-    def _descriptionsMalop(self):
-        if (len(self.DESC_MALOP_TRANSLATION) > 0):
-            return self.DESC_MALOP_TRANSLATION
-
-        url = f"{self.host}/{self.DESCRIPTION_MALOP_URI}"
-        data = self.execute_query("GET", url, None)
-        self.DESC_MALOP_TRANSLATION = data
-        return data
+            return False, str(e)
 
-    def _descriptionsFeatures(self):
-        # retrive the traslation table for suspicion and other info from malop
-        if (len(self.DESC_TRANSLATION) > 0):
-            return self.DESC_TRANSLATION
+    def getComments(self, alertMalopId):
 
-        url = f"{self.host}/{self.DESCRIPTION_URI}"
-        data = self.execute_query("GET", url, None)
-        self.DESC_TRANSLATION = data
-        return data
+        url_alert_malop_comments = self.host + '/' + self.ALERT_MALOP_COMMENT_URI
 
-    def getDevicesDetails(self, devices_ids):
-        sensors_url = f"{self.host}/{self.SENSOR_URI}"
-        query = {
-            'sortDirection': 'ASC',
-            'sortingFieldName': 'machineName',
-            'filters': [
-                {'fieldName': "machineName", 'operator': "ContainsIgnoreCase", 'values': devices_ids},
-                {"fieldName": "status", "operator": "NotEquals", "values": ["Archived"]}
-            ]
+        header = {
+            'Content-Type': 'text/plain'
         }
-        data = self.execute_query("POST", sensors_url, query)
-        return data.get('sensors', [])
+        return self.execute_query(method="POST", url=url_alert_malop_comments, data=alertMalopId, header=header)
 
     def suspicions(self, malop_id):
-        self._descriptionsFeatures()
+
+        if not (len(self.DESC_TRANSLATION) > 0):
+            url = self.host + "/" + self.DESCRIPTION_URI
+            self.DESC_TRANSLATION = self.execute_query("GET", url)
 
         status, suspicions_process = self.suspicionsProcess(malop_id)
         if not status:
```

Comment:
might want to prefix functions `suspicions()`, `suspicionsProcess()` and `suspicionsNetwork()` with `get` to respect other functions' nomenclature and clarity of code